### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by matiasbertani**
I add a little patch to the method prepararion_cuentas that
does not affect the behavior only used to return the names of the
result file it creates.

To run the integration test while being at the root of the project
run the following command
```bash 
python -m tests.integration_tests
```

### CODE REVIEW

**Review by natilou — APPROVED** [review]
Great integration test!

**Comment by matiasbertani** [comment]
I made the rebase, but this PR now has the error from #7, so we need to wait until #12  be merged

@natilou  could you review my code again?

**Review by natilou — APPROVED** [review]
LGTM

**Pull request merged**
